File: 1. Two Sum/two sum.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* twoSum(int* nums, int numsSize, int target) {
    int *arr = malloc(2*sizeof(int));
    int flag = 0;
    for(int i = 0; i < numsSize; i++ )
    {
        for(int j = i+1; j < numsSize; j++)
        {
            if(nums[j] == target - nums[i])
            {
                arr[0] = i;
                arr[1] = j;
                flag = 1;
                break;
            }
        }
        if(flag == 1)
        {
            break;
        }
    }
    return arr;
}

File: 1. Two Sum/two_sum.py
#Hashmap solution
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashmap = {} # val:index
        for i, num in enumerate(nums):
            if (target - num) in hashmap:
                return [hashmap[target-num],i]
            hashmap[num] = i
        
